concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+Lastra-1986-381-21	21	llovizna	drizzle	1016	DRIZZLE
+Lastra-1986-381-22	22	relámpago	flash of lightning	1654	LIGHTNING
+Lastra-1986-381-23	23	trueno	thunder	1150	THUNDER
+Lastra-1986-381-24	24	nube	cloud	1489	CLOUD
+Lastra-1986-381-25	25	neblina	fog	249	FOG
+Lastra-1986-381-26	26	cabeza (sin posesivo)	Head (without possessive)	1256	HEAD
+Lastra-1986-381-27	27	cabello (sin posesivo)	hair (without possessive)	2648	HAIR (HEAD)
+Lastra-1986-381-28	28	cara	face	1560	FACE
+Lastra-1986-381-29	29	frente	forehead	123	FOREHEAD
+Lastra-1986-381-30	30	mi ojo	my eye	1248	EYE
+Lastra-1986-381-31	31	ceja	eyebrow	1173	EYEBROW
+Lastra-1986-381-32	32	mi nariz	my nose	1221	NOSE
+Lastra-1986-381-33	33	oreja (sin posesivo)	ear (without possessive)	1247	EAR
+Lastra-1986-381-34	34	mi boca	my mouth	674	MOUTH
+Lastra-1986-381-35	35	boca (sin posesivo)	mouth (without possessive)	674	MOUTH
+Lastra-1986-381-36	36	labio	lip	478	LIP
+Lastra-1986-381-37	37	lengua (sin posesivo)	tongue (without possessive)	1205	TONGUE
+Lastra-1986-381-38	38	diente (sin posesivo)	tooth (without possessive)	1380	TOOTH
+Lastra-1986-381-39	39	encía	gum		
+Lastra-1986-381-40	40	cuello	neck	1333	NECK
+Lastra-1986-381-41	41	hombro	shoulder	1482	SHOULDER
+Lastra-1986-381-42	42	espalda	back	1291	BACK
+Lastra-1986-381-43	43	cintura	waist	1681	WAIST
+Lastra-1986-381-44	44	seno (en singular)	breast (singular)	1402	BREAST
+Lastra-1986-381-45	45	barriga	belly	1251	BELLY
+Lastra-1986-381-46	46	ombligo (sin posesivo)	navel (without possessive)	1838	NAVEL
+Lastra-1986-381-47	47	nalgas	buttocks	834	BUTTOCKS
+Lastra-1986-381-48	48	mano (sin posesivo)	hand (without possessive)	1277	HAND
+Lastra-1986-381-49	49	tu mano	your hand		
+Lastra-1986-381-50	50	uña (sin posesivo)	NAIL (without possessive)	1896	FINGERNAIL OR TOENAIL
+Lastra-1986-381-51	51	mi uña	my nail		
+Lastra-1986-381-52	52	codo	elbow	981	ELBOW
+Lastra-1986-381-53	53	pierna (especificar de dónde a dónde la llaman así)	leg (specify from where to where they called it like this)	1297	LEG
+Lastra-1986-381-54	54	rodilla (especificar si distinguen la 'rótula')	knee (specify whether they distinguish the ""kneecap"")	1371	KNEE
+Lastra-1986-381-55	55	mi pie	my foot	1301	FOOT
+Lastra-1986-381-56	56	corazón	heart	1223	HEART
+Lastra-1986-381-57	57	hígado	liver	1224	LIVER
+Lastra-1986-381-58	58	mi hueso	My bone	1394	BONE
+Lastra-1986-381-59	59	piel (de una persona)	skin (of a person)	2613	SKIN (HUMAN)
+Lastra-1986-381-60	60	sangre	blood	946	BLOOD
+Lastra-1986-381-61	61	mi sangre	my blood		
+Lastra-1986-381-62	62	saliva	saliva	1503	SALIVA
+Lastra-1986-381-63	63	lágrima	tear	1651	TEAR (OF EYE)
+Lastra-1986-381-64	64	pus	pus	1808	PUS
+Lastra-1986-381-65	65	lagaña	lagaña		
+Lastra-1986-381-66	66	caspa	dandruff	576	DANDRUFF
+Lastra-1986-381-67	67	sarna	scabies		
+Lastra-1986-381-68	68	viruela	smallpox		
+Lastra-1986-381-69	69	su cuerno	his horn	1393	HORN (ANATOMY)
+Lastra-1986-381-70	70	pico (de un pájaro)	peak (of a bird)	73	BEAK
```

Comment:
Translation: `beak (of a bird)`?

Reply:
Yes, thanks for the catch, this slipped out of my sight too.